### PR TITLE
added higher food and just food checking

### PR DESCRIPTION
**Pull request by yajinni**
image

### CODE REVIEW

File: src/parser/shared/modules/features/Checklist2/PreparationRule.js [hunk 1]
```diff
@@ -58,12 +58,17 @@ class PreparationRule extends React.PureComponent {
     return (
       <>
         <Requirement
-          name="Food used"
-          thresholds={thresholds.foodPresent}
+          name="High Quality Food used"
+          thresholds={thresholds.higherfoodPresent}
```
Comment (Pewtro) [inline src/parser/shared/modules/features/Checklist2/PreparationRule.js]: Name="High quality food used"

**Comment by yajinni** [comment]
http://localhost:3000/report/kXJdwmhqD3z4frna/1-Heroic+Taloc+-+Kill+(7:09)/17-Tattoomoon

File: src/parser/shared/modules/items/FoodChecker.js [hunk 2]
```diff
@@ -41,19 +41,26 @@ class FoodChecker extends Analyzer {
...
   suggestions(when) {
     let suggestionText = 'You did not have any food buff combat started. Having the right food buff during combat is an easy way to improve performance.';
     if (!this.higherFoodUp && this.lowerFoodUp) {
       suggestionText = 'You did not have the best food active when starting the fight. Using the best food available is an easy way to improve performance.';
     }
-    when(this.foodSuggestionThresholds)
+    when(this.higherFoodSuggestionThresholds)
       .addSuggestion((suggest) => {
         return suggest(suggestionText)
           .icon(SPELLS.BOUNTIFUL_CAPTAIN_FEAST_AGI.icon)
```
Comment (Pewtro) [inline src/parser/shared/modules/items/FoodChecker.js]: Should probably add an average or major issue for no food at all?

Comment (yajinni) [reply src/parser/shared/modules/items/FoodChecker.js]: good idea

File: src/parser/shared/modules/features/Checklist2/PreparationRule.js (hunk 1 above)
Comment (yajinni) [reply src/parser/shared/modules/features/Checklist2/PreparationRule.js]: ah yes my bad

File: src/parser/shared/modules/items/FoodChecker.js (hunk 2 above)
Comment (yajinni) [reply src/parser/shared/modules/items/FoodChecker.js]: http://localhost:3000/report/kXJdwmhqD3z4frna/1-Heroic+Taloc+-+Kill+(7:09)/12-Deathsblad%C3%A9

image

File: src/parser/shared/modules/features/Checklist2/PreparationRule.js (hunk 1 above)
Comment (Pewtro) [reply src/parser/shared/modules/features/Checklist2/PreparationRule.js]: Food is still capitalized 😢

File: src/parser/shared/modules/items/FoodChecker.js [hunk 3]
```diff
@@ -56,15 +56,20 @@ class FoodChecker extends Analyzer {
     };
   }
   suggestions(when) {
+    let importance = SUGGESTION_IMPORTANCE.MINOR;
     let suggestionText = 'You did not have any food buff combat started. Having the right food buff during combat is an easy way to improve performance.';
     if (!this.higherFoodUp && this.lowerFoodUp) {
```
Comment (Pewtro) [inline src/parser/shared/modules/items/FoodChecker.js]: this suggestionText is never used as it is right now.

File: src/parser/shared/modules/items/FoodChecker.js [hunk 4]
```diff
@@ -56,15 +56,20 @@ class FoodChecker extends Analyzer {
...
       suggestionText = 'You did not have the best food active when starting the fight. Using the best food available is an easy way to improve performance.';
     }
+    if (!this.higherFoodUp && !this.lowerFoodUp) {
+      suggestionText = 'You did not any food active when starting the fight. Using food is an easy way to improve performance.';
+      importance = SUGGESTION_IMPORTANCE.MAJOR;
```
Comment (Pewtro) [inline src/parser/shared/modules/items/FoodChecker.js]: You did not **HAVE**

File: src/parser/shared/modules/items/FoodChecker.js (hunk 3 above)
Comment (Pewtro) [reply src/parser/shared/modules/items/FoodChecker.js]: There's 2 spaces here now 😅

File: src/parser/shared/modules/features/Checklist2/PreparationRuleAnalyzer.js [hunk 5]
```diff
@@ -30,7 +30,8 @@ class PreparationRuleAnalyzer extends Analyzer {
         style: 'number',
       },
       flaskPresent: this.flaskChecker.flaskSuggestionThresholds,
-      foodPresent: this.foodChecker.foodSuggestionThresholds,
+      higherfoodPresent: this.foodChecker.higherFoodSuggestionThresholds,
+      foodPresent: this.foodChecker.isPresentFoodSuggestionThresholds,
```
Comment (Pewtro) [inline src/parser/shared/modules/features/Checklist2/PreparationRuleAnalyzer.js]: This should be higherFoodPresent

**Pull request merged**